internal/storage/util.go

```diff
@@ -27,7 +20,7 @@ func BatchLoadPolicy(
 ) error {
 	for idx := range ids {
 		if idx%maxPoliciesInBatch == 0 {
-			idxEnd := minInt(idx+maxPoliciesInBatch, len(ids))
+			idxEnd := min(idx+maxPoliciesInBatch, len(ids))
```

Comment:
Go 1.21 added a new built-in `min` function so we don't need the `minInt` helper function anymore

https://go.dev/ref/spec#Min_and_max